### PR TITLE
Fixing Copy error in code

### PR DESCRIPTION
**Pull request by MohamedMafaz**
Fixed Code Copying Error

I fixed the page that copy code with explanation. But It's fixed now

_Issues fixed by this PR (if any):_ Fixes #7504

### CODE REVIEW

**Comment by domesticmouse** [comment]
I'm guessing you are attempting to make it possible to use the cut/paste affordance, however this change is insufficient. Cutting and pasting from here will still copy the `$` at the start of the command.

File: src/get-started/codelab.md [hunk 1]
```diff
@@ -226,6 +226,8 @@ as well as many other open source packages, on [pub.dev][].
 
     ```terminal
     $ flutter pub add english_words
```
Comment (sfshaza2) [inline src/get-started/codelab.md:228]:
```suggestion
$ flutter pub add english_words
```

File: src/get-started/codelab.md [hunk 2]
```diff
@@ -258,6 +260,8 @@ as well as many other open source packages, on [pub.dev][].
 
     ```terminal
     $ flutter pub get
```
Comment (sfshaza2) [inline src/get-started/codelab.md:262]:
```suggestion
$ flutter pub get
```

**Review by sfshaza2 — APPROVED** [review]
lgtm, with some changes

**Comment by sfshaza2** [comment]
@MohamedMafaz, I can't merge this until you sign the CLA.

**Comment by MohamedMafaz** [comment]
Sir, I signed for CLA

File: src/get-started/codelab.md [hunk 3]
```diff
@@ -225,7 +225,12 @@ as well as many other open source packages, on [pub.dev][].
  1. Add `english_words` package to your project as follows:
 
     ```terminal
-    $ flutter pub add english_words
+$ flutter pub add english_words
+    ```
+
+The output will look something like the following:
```
Comment (domesticmouse) [inline src/get-started/codelab.md:231]: Will this de-dent break the list?

File: src/get-started/codelab.md [hunk 4]
```diff
@@ -225,7 +225,12 @@ as well as many other open source packages, on [pub.dev][].
  1. Add `english_words` package to your project as follows:
 
     ```terminal
-    $ flutter pub add english_words
+$ flutter pub add english_words
```
Comment (domesticmouse) [inline src/get-started/codelab.md:228]: I don't believe this de-dent is required.

Comment (MohamedMafaz) [reply src/get-started/codelab.md]: Sir, I corrected it. Please Check it out

**Comment by sfshaza2** [comment]
I manually confirm that he has signed.

**Pull request merged**